src/javascripts/ng-admin/Crud/column/maColumn.js

```diff
@@ -51,12 +51,16 @@ export default function maColumn($state, $anchorScroll, $compile, Configuration,
                     element.append(FieldViewConfiguration[type].getReadWidget());
                 }
             }
-            scope.detailState = getDetailLinkRouteName(scope.field, scope.entity);
-            scope.detailStateParams = {
-                ...$state.params,
-                entity: scope.entry.entityName,
-                id: scope.entry.identifierValue,
-            };
+            scope.detailState = false;
+            scope.detailStateParams = {};
+            if (angular.isDefined(scope.entry)) {
```

Comment:
please use `typeof scope.entry === 'undefined'` instead to avoid the dependency on the `angular` global